### PR TITLE
pd-control: add scheduler status

### PR DESCRIPTION
**Pull request by CabinfeverB**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add the description of `scheduler describe balance-region-scheduler`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/tikv/pd/issues/5257

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -1110,8 +1110,15 @@ region check miss-peer
...
 >> scheduler resume all                                   // 继续运行所有的调度器
 >> scheduler config balance-hot-region-scheduler          // 显示 balance-hot-region 调度器的配置
+>> scheduler describe balance-region-scheduler            // 显示 balance-region 的运行状态和相应的诊断信息
 ```
 
+### `scheduler describe balance-region-scheduler`
+
+用于查看 balance-region 调度器的运行状态和相应的诊断信息。运行状态有五种类型，包括 `disabled`、`paused`、`scheduling`、`pending`、`normal`。`disabled` 表示当前调度器不可用或是被移除，`paused` 表示当前调度器暂停工作，`scheduling` 表示当前调度器正在生成或执行调度、`pending` 表示当前调度器无法产生调度，`normal` 表示当前调度器无需进行调度。对于 `pending` 状态的 scheduler ，还会返回一个概览信息，来帮助用户诊断。概览信息包含了 store 的一些状态信息，解释了它们为什么不能被选中进行调度。
+
+从 TiDB v6.3.0 起，PD 为 `balance-region-scheduler` 提供了运行状态和简要诊断信息的功能，其余 scheduler 和 check 暂未支持。
```
Comment (rleungx) [inline pd-control.md:1120]: checker

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
@rleungx PTAL again

File: pd-control.md [hunk 2]
```diff
@@ -1110,8 +1110,23 @@ region check miss-peer
...
 >> scheduler resume all                                   // 继续运行所有的调度器
 >> scheduler config balance-hot-region-scheduler          // 显示 balance-hot-region 调度器的配置
+>> scheduler describe balance-region-scheduler            // 显示 balance-region 的运行状态和相应的诊断信息
 ```
 
+### `scheduler describe balance-region-scheduler`
+
+用于查看 `balance-region-scheduler` 的运行状态和相应的诊断信息。
+
+从 TiDB v6.3.0 起，PD 为 `balance-region-scheduler` 提供了运行状态和简要诊断信息的功能，其余 scheduler 和 checker 暂未支持。
```
Comment (rleungx) [inline pd-control.md:1120]: Does it support balance-leader?

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: pd-configuration-file.md [hunk 3]
```diff
@@ -285,6 +285,11 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 是否使用 Joint Consensus 进行副本调度。关闭该特性时，PD 将采用一次调度一个副本的方式进行调度。
 + 默认值：true
 
+### `enable-diagnosis` <span class="version-mark">从 v6.3 版本开始引入</span>
+
++ 是否开启诊断功能。开启特性时，PD 将会记录调度中的一些状态来帮助诊断。开启时会略微影响调度速度，在 Store 数量较多时会消耗较大内存。
++ 默认值：false
+
```
Comment (Oreoxmt) [inline pd-configuration-file.md:292]:
```suggestion
### `enable-diagnosis` <span class="version-mark">从 v6.3.0 版本开始引入</span>

+ 是否开启诊断功能。开启特性时，PD 将会记录调度中的一些状态来帮助诊断。开启时会略微影响调度速度，在 Store 数量较多时会消耗较大内存。
+ 默认值：false

```

File: pd-control.md [hunk 4]
```diff
@@ -1110,8 +1110,23 @@ region check miss-peer
...
 >> scheduler resume all                                   // 继续运行所有的调度器
 >> scheduler config balance-hot-region-scheduler          // 显示 balance-hot-region 调度器的配置
+>> scheduler describe balance-region-scheduler            // 显示 balance-region 的运行状态和相应的诊断信息
 ```
 
+### `scheduler describe balance-region-scheduler`
+
+用于查看 `balance-region-scheduler` 的运行状态和相应的诊断信息。
+
+从 TiDB v6.3.0 起，PD 为 `balance-region-scheduler` 和 `balance-leader-scheduler` 提供了运行状态和简要诊断信息的功能，其余 scheduler 和 checker 暂未支持。你可以通过 pd-ctl 修改 `enable-diagnosis` 配置项开启该功能。
```
Comment (Oreoxmt) [inline pd-control.md:1120]:
```suggestion
从 TiDB v6.3.0 起，PD 为 `balance-region-scheduler` 和 `balance-leader-scheduler` 提供了运行状态和简要诊断信息的功能，其余 scheduler 和 checker 暂未支持。你可以通过 `pd-ctl` 修改 [`enable-diagnosis`](/pd-configuration-file.md#enable-diagnosis-从-v630-版本开始引入) 配置项开启该功能。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: pd-configuration-file.md (hunk 3 above)
Comment (rleungx) [reply pd-configuration-file.md:292]: Should it be diagnosis or diagnostic? @Oreoxmt

Comment (Oreoxmt) [reply pd-configuration-file.md:292]: > Should it be diagnosis or diagnostic? @Oreoxmt

IMO, it should be diagnostics. This configuration item provides details about scheduler status, which helps user to diagnosis.

File: pd-control.md [hunk 5]
```diff
@@ -1110,8 +1110,23 @@ region check miss-peer
...
 >> scheduler resume all                                   // 继续运行所有的调度器
 >> scheduler config balance-hot-region-scheduler          // 显示 balance-hot-region 调度器的配置
+>> scheduler describe balance-region-scheduler            // 显示 balance-region 的运行状态和相应的诊断信息
 ```
 
+### `scheduler describe balance-region-scheduler`
+
+用于查看 `balance-region-scheduler` 的运行状态和相应的诊断信息。
+
+从 TiDB v6.3.0 起，PD 为 `balance-region-scheduler` 和 `balance-leader-scheduler` 提供了运行状态和简要诊断信息的功能，其余 scheduler 和 checker 暂未支持。你可以通过 `pd-ctl` 修改 [`enable-diagnosis`](/pd-configuration-file.md#enable-diagnosis-从-v630-版本开始引入) 配置项开启该功能。
```
Comment (Oreoxmt) [inline pd-control.md:1120]:
```suggestion
从 TiDB v6.3.0 起，PD 为 `balance-region-scheduler` 和 `balance-leader-scheduler` 提供了运行状态和简要诊断信息的功能，其余 scheduler 和 checker 暂未支持。你可以通过 `pd-ctl` 修改 [`enable-diagnostic`](/pd-configuration-file.md#enable-diagnosis-从-v630-版本开始引入) 配置项开启该功能。
```

**Review by Oreoxmt — APPROVED** [review]

File: pd-control.md [hunk 6]
```diff
@@ -1110,8 +1110,23 @@ region check miss-peer
...
 >> scheduler resume all                                   // 继续运行所有的调度器
 >> scheduler config balance-hot-region-scheduler          // 显示 balance-hot-region 调度器的配置
+>> scheduler describe balance-region-scheduler            // 显示 balance-region 的运行状态和相应的诊断信息
 ```
 
+### `scheduler describe balance-region-scheduler`
+
+用于查看 `balance-region-scheduler` 的运行状态和相应的诊断信息。
+
+从 TiDB v6.3.0 起，PD 为 `balance-region-scheduler` 和 `balance-leader-scheduler` 提供了运行状态和简要诊断信息的功能，其余 scheduler 和 checker 暂未支持。你可以通过 `pd-ctl` 修改 [`enable-diagnostic`](/pd-configuration-file.md#enable-diagnosis-从-v630-版本开始引入) 配置项开启该功能。
```
Comment (Oreoxmt) [inline pd-control.md:1120]:
```suggestion
从 TiDB v6.3.0 起，PD 为 `balance-region-scheduler` 和 `balance-leader-scheduler` 提供了运行状态和简要诊断信息的功能，其余 scheduler 和 checker 暂未支持。你可以通过 `pd-ctl` 修改 [`enable-diagnostic`](/pd-configuration-file.md#enable-diagnostic-从-v630-版本开始引入) 配置项开启该功能。
```

**Review by Oreoxmt — APPROVED** [review]

**Comment by Oreoxmt** [comment]
@rleungx PTAL

**Comment by Oreoxmt** [comment]
@rleungx PTAL

File: pd-control.md [hunk 7]
```diff
@@ -1110,8 +1110,23 @@ region check miss-peer
...
+
+用于查看 `balance-region-scheduler` 的运行状态和相应的诊断信息。
+
+从 TiDB v6.3.0 起，PD 为 `balance-region-scheduler` 和 `balance-leader-scheduler` 提供了运行状态和简要诊断信息的功能，其余 scheduler 和 checker 暂未支持。你可以通过 `pd-ctl` 修改 [`enable-diagnostic`](/pd-configuration-file.md#enable-diagnostic-从-v630-版本开始引入) 配置项开启该功能。
+
+调度器运行状态有以下几种类型：
+
+- `disabled`：表示当前调度器不可用或被移除。
+- `paused`：表示当前调度器暂停工作。
+- `scheduling`：表示当前调度器正在生成。
```
Comment (rleungx) [inline pd-control.md:1126]:
```suggestion
- `scheduling`：表示当前调度器正在生成调度。
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**